src/openrct2/peep/peep.c

```diff
@@ -1730,37 +1638,37 @@ typedef struct item_pref {
 } item_pref;
 
 item_pref item_order_preference[] = {
-		{ 0, PEEP_ITEM_ICE_CREAM, 15 },
-		{ 0, PEEP_ITEM_FRIES, 16},
-		{ 0, PEEP_ITEM_PIZZA, 22 },
-		{ 0, PEEP_ITEM_BURGER, 17 },
-		{ 0, PEEP_ITEM_DRINK, 18 },
-		{ 0, PEEP_ITEM_COFFEE, 35 },
-		{ 0, PEEP_ITEM_CHICKEN, 34 },
-		{ 0, PEEP_ITEM_LEMONADE, 37 },
-		{ 0, PEEP_ITEM_COTTON_CANDY, 20 },
-		{ 0, PEEP_ITEM_POPCORN, 22 },
-		{ 0, PEEP_ITEM_HOT_DOG, 31 },
-		{ 0, PEEP_ITEM_TENTACLE, 32 },
-		{ 0, PEEP_ITEM_CANDY_APPLE, 33 },
-		{ 0, PEEP_ITEM_DONUT, 34 },
-		{ 1, PEEP_ITEM_PRETZEL, 39 },
-		{ 1, PEEP_ITEM_COOKIE, 39 },
-		{ 1, PEEP_ITEM_CHOCOLATE, 35 },
-		{ 1, PEEP_ITEM_ICED_TEA, 35 },
-		{ 1, PEEP_ITEM_FUNNEL_CAKE, 43 },
-		{ 1, PEEP_ITEM_BEEF_NOODLES, 44 },
-		{ 1, PEEP_ITEM_FRIED_RICE_NOODLES, 44 },
-		{ 1, PEEP_ITEM_WONTON_SOUP, 46 },
-		{ 1, PEEP_ITEM_MEATBALL_SOUP, 46 },
-		{ 1, PEEP_ITEM_FRUIT_JUICE, 43 },
-		{ 1, PEEP_ITEM_SOYBEAN_MILK, 41 },
-		{ 1, PEEP_ITEM_SU_JONGKWA, 41 },
-		{ 1, PEEP_ITEM_SUB_SANDWICH, 47 },
-		{ 1, PEEP_ITEM_ROAST_SAUSAGE, 45 },
-		{ 0, PEEP_ITEM_BALLOON, 19 },
-		{ 0, PEEP_ITEM_HAT, 30},
-		{ 1, PEEP_ITEM_SUNGLASSES, 40},
+		{ 0, PEEP_ITEM_ICE_CREAM, PEEP_SPRITE_TYPE_ICE_CREAM },
+		{ 0, PEEP_ITEM_FRIES, PEEP_SPRITE_TYPE_FRIES },
+		{ 0, PEEP_ITEM_PIZZA, PEEP_SPRITE_TYPE_PIZZA },
+		{ 0, PEEP_ITEM_BURGER, PEEP_SPRITE_TYPE_BURGER },
+		{ 0, PEEP_ITEM_DRINK, PEEP_SPRITE_TYPE_DRINK },
+		{ 0, PEEP_ITEM_COFFEE, PEEP_SPRITE_TYPE_COFFEE },
+		{ 0, PEEP_ITEM_CHICKEN, PEEP_SPRITE_TYPE_CHICKEN },
+		{ 0, PEEP_ITEM_LEMONADE, PEEP_SPRITE_TYPE_LEMONADE },
+		{ 0, PEEP_ITEM_COTTON_CANDY, PEEP_SPRITE_TYPE_CANDYFLOSS },
+		{ 0, PEEP_ITEM_POPCORN, PEEP_SPRITE_TYPE_PIZZA },
+		{ 0, PEEP_ITEM_HOT_DOG, PEEP_SPRITE_TYPE_HOT_DOG  },
+		{ 0, PEEP_ITEM_TENTACLE, PEEP_SPRITE_TYPE_TENTACLE },
+		{ 0, PEEP_ITEM_CANDY_APPLE, PEEP_SPRITE_TYPE_TOFFEE_APPLE },
+		{ 0, PEEP_ITEM_DONUT, PEEP_SPRITE_TYPE_DONUT },
+		{ 1, PEEP_ITEM_PRETZEL, PEEP_SPRITE_TYPE_PRETZEL },
+		{ 1, PEEP_ITEM_COOKIE, PEEP_SPRITE_TYPE_PRETZEL },
+		{ 1, PEEP_ITEM_CHOCOLATE, PEEP_SPRITE_TYPE_COFFEE },
+		{ 1, PEEP_ITEM_ICED_TEA, PEEP_SPRITE_TYPE_COFFEE },
+		{ 1, PEEP_ITEM_FUNNEL_CAKE, PEEP_SPRITE_TYPE_FUNNEL_CAKE },
+		{ 1, PEEP_ITEM_BEEF_NOODLES, PEEP_SPRITE_TYPE_NOODELS },
```

Comment:
@duncanspumpkin There are various VS extensions, but they can hurt performance sometimes or just be annoying with the amount of technical terms which aren't officially words.